input/fsh/DEF_Operations.fsh

```diff
@@ -1,9 +1,8 @@
-Instance: Operation-mcode-patient-everything
+Instance: mcode-patient-everything
```

Comment:
While it makes sense to do this change, this goes back to a naming convention in the style-guide. The upside of specifying "Operation" in the name is that the implementer quickly knows what it is. The downside is that it's generally not a good convention to call every definition by its type.

Reply:
This was the only case of ALL instances in mCODE where the `id` diverged from the name. In an instance, the name and the id are the same (there really is no name). So in a way, the way it was is self-contradictory.